<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1525</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1521</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion des El√®ves</string>
     </attribute>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1571</width>
        <height>691</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>291</width>
          <height>481</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>Ajouter Eleve</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Prenom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>160</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Date de Naissance</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_Prenom_ray">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id_ray">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom_ray">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>80</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>80</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>280</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_modif_ray">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>350</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>modifier</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_supp_ray">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>280</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_afficher_ray">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>350</y>
           <width>91</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>afficher</string>
         </property>
        </widget>
        <widget class="QDateEdit" name="dateEdit_naissance">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>160</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_pdf_ray">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>420</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>generer pdf</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_stat_ray">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>420</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>statistiques</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>210</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Enseignant</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id_ray_el_en">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>210</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QTableView" name="tableView_ray">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>60</y>
          <width>521</width>
          <height>251</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>340</y>
          <width>241</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>recherche</string>
        </property>
        <widget class="QLineEdit" name="lineEdit_rech_id_ray">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_rech_nom_ray">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_rech_prenom_ray">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>110</y>
           <width>113</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>70</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>NOM</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>110</y>
           <width>56</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>PRENOM</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>330</y>
          <width>181</width>
          <height>171</height>
         </rect>
        </property>
        <property name="title">
         <string>TRI</string>
        </property>
        <widget class="QPushButton" name="pushButton_triID_ray">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>30</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>TRI_ID</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_trinom_ray">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>70</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>TRI_NOM</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_triprenom_ray">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>110</y>
           <width>93</width>
           <height>28</height>
          </rect>
         </property>
         <property name="text">
          <string>TRI_PRENOM</string>
         </property>
        </widget>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>890</x>
          <y>70</y>
          <width>331</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des Enseignants</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>291</width>
        <height>481</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter Enseignants</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>160</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date de Naissance</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Prenom_ray_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_id_ray_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_ray_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modif_ray_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>modifier</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supp_ray_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>280</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>supprimer</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_afficher_ray_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>340</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>afficher</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_naissance_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>200</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Salaire</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Salaire_ray">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>200</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_pdf_ray_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>generer pdf</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stat_ray_4">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>400</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>statistiques</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_gendre_ray_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>240</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Masculin</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feminin</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Gendre</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_ray_2">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>571</width>
        <height>251</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>241</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>recherche</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_rech_id_ray_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rech_nom_ray_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Id</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_rech_prenom_ray_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>330</y>
        <width>181</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>TRI</string>
      </property>
      <widget class="QPushButton" name="pushButton_triID_ray_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>TRI_ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_trinom_ray_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>TRI_NOM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_trisalaire_ray_2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>110</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>TRI_SALAIRE</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1525</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
